static/js/stream_ui_updates.js

```diff
@@ -93,6 +93,22 @@ exports.update_regular_sub_settings = function (sub) {
     }
 };
 
+exports.update_stream_properties_edit_elements = function (sub) {
+    if (!stream_edit.is_sub_settings_active(sub)) {
+        return;
+    }
+    const $settings = $(".subscription_settings[data-stream-id='" + sub.stream_id + "']");
+    if (sub.can_perform_admin_actions) {
+        $settings.find(".stream-name .editable").show();
+        $settings.find(".stream-description .editable").show();
+        $settings.find(".deactivate").show();
```

Comment:
I will try to represent all three elements simultaneously using a class (a new or common one).
(Although this code is likely to change by one of my PR in the future.)

Reply:
We can add a new class here like stream-settings-header-controls.